sig-services/README.md

```diff
@@ -0,0 +1,55 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+This file is part of https://github.com/open-services-group/community
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Services Special Interest Group
+
+Oversee, steer and organize involvement and commitment of new and existing services.
+
+The [charter](charter.md) defines the scope and governance of the Services Special Interest Group.
+
+## Meetings
+* Regular SIG Meeting: [TBDs at TBD UTC](TBD) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=TBD&tz=UTC).
+
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+
+* Tom Coufal (**[@tumido](https://github.com/tumido)**), Red Hat
+
+## Contact
+- [Mailing list]()
+- [Open Community Issues/PRs](https://github.com/open-services-group/community/labels/sig%2Fservices)
+- Steering Committee Liaison: Tom Coufal (**[@tumido](https://github.com/tumido)**)
+
+## Subprojects
+
+The following [subprojects][subproject-definition] are owned by sig-services:
+### AICoE-CD
```

Comment:
I'm ok with the list of subprojects for now, though this'll probably change after the first few sig meetings